UnityProject/Assets/Scripts/UI/Systems/MainHUD/UI Bottom/HandsController.cs

```diff
@@ -83,21 +86,57 @@ public void AddHand(IDynamicItemSlotS bodyPartUISlots, BodyPartUISlots.StorageCh
 					DoubleHandControllers.Add(HandController);
 				}
 
+				break;
+			case NamedSlot.storage01:
+			case NamedSlot.storage02:
+			case NamedSlot.storage03:
+			case NamedSlot.storage04:
+			case NamedSlot.storage05:
+			case NamedSlot.storage06:
+			case NamedSlot.storage07:
+			case NamedSlot.storage08:
+			case NamedSlot.storage09:
+			case NamedSlot.storage10:
```

Comment:
what in sam hell is this?

Reply:
that's collapsing the handling of all those cases in a body

Reply:
yes but i mean why do this
cant you just use objects instead of enums?

Reply:
it's for the code that handles the active hand

Reply:
> cant you just use objects instead of enums?

elaborate

Reply:
public partial class PlayerNetworkActions : NetworkBehaviour
{

	[NonSerialized] public GameObject activeHand;
	[NonSerialized] public NamedSlot CurrentActiveHand = NamedSlot.rightHand;